.markdownlint.json

```diff
@@ -0,0 +1,12 @@
+{
+  "default": true,
+  "MD004": false,
+  "MD013": false,
+  "MD025": false,
+  "MD026": false,
+  "MD029": false,
+  "MD036": false,
```

Comment:
Another decent idea. Headings at least give link anchors. The only place where this was used was `CICD.md` and the emphasized list items can easily be converted to headings.